3D-microservices-observability/node-js-app/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "beachshirt",
+  "version": "1.0.0",
+  "description": "beachshirt application which makes cool shirts for the beach.",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Yogesh Prasad",
```

Comment:
You shouldn't just delete the license line from a package.  I've been told that the rest of our stuff is Apache 2, so you should use that.